crates/router/src/connector/rapyd.rs

```diff
@@ -881,7 +879,7 @@ impl api::IncomingWebhook for Rapyd {
         }?;
         Ok(api::disputes::DisputePayload {
             amount: webhook_dispute_data.amount.to_string(),
-            currency: webhook_dispute_data.currency,
+            currency: webhook_dispute_data.currency.to_string(),
```

Comment:
@sai-harsha-vardhan Can the `currency` field in `DisputePayload` be the `Currency` enum, instead of `String`?

Reply:
Yes, it is feasible to use the Currency enum as the data type for currency field

Reply:
https://github.com/juspay/hyperswitch-cloud/issues/452